website/docs/index.html.markdown

```diff
@@ -246,22 +246,181 @@ provider "vault" {
 }
 ```
 
-## Example Multiple Namespace Support
-To leverage more than one namespace in Vault you can use a Terraform alias:
+## Namespace support 
+
+The Vault provider supports managing [Namespaces][namespaces] (a feature of
+Vault Enterprise), as well as creating resources in those namespaces by
+utilizing [Provider Aliasing][aliasing]. The `namespace` option in the [provider
+block][provider-block] enables the management of  resources in the specified
+namespace. 
+
+### Using Provider Aliases
+
+The below configuration is a simple example of using the provider block's
+`namespace` attribute to configure an aliased provider and create a resource
+within that namespace:
 
 ```hcl
-provider "vault" {
-  alias = "ns1"
-  namespace = "ns1"
+ # main provider block with no namespace
+provider vault {}
+
+ # create the "everyone" namespace in the default root namespace
+resource "vault_namespace" "everyone" {
+  path = "everyone"
 }
 
-provider "vault" {
-  alias = "ns2"
-  namespace = "ns2"
+ # configure an aliased provider, scope to the new namespace. Note: at time of
+ # writing, provider configuration does not support the depends_on option.
+provider vault {
+  alias     = "everyone"
+  namespace = "everyone"
 }
 
-resource "vault_generic_secret" "secret"{
-  provider = "vault.ns1"
-  ...
+ # create a policy in the "everyone" namespace
+resource "vault_policy" "example" {
+  provider = vault.everyone
+
+  depends_on = [vault_namespace.everyone]
+  name       = "vault_everyone_policy"
+  policy     = data.vault_policy_document.list_secrets.hcl
 }
+
+data "vault_policy_document" "list_secrets" {
+  rule {
+    path         = "secret/*"
+    capabilities = ["list"]
+    description  = "allow List on secrets under everyone/"
+  }
+}
+```
+
+Using this alias configuration, the policy `list_secrets` is created under the
+`everyone` namespace, but not under the "root" namespace:
+
+```
+$ vault policy list -namespace=everyone
+default
+vault_everyone_policy
+
+$ vault policy list
+default
+root
 ```
+
+### Nested Namespaces
+
+A more complex example of nested namespaces is possible using provider aliases
+and the `depends_on` configuration option. At time of writing Terraform (v0.13)
+does not support using the `depends_on` attribute in the `provider` block, which
+prevents the normal 
```

Comment:
Incomplete sentence?

Reply:
Good catch, thanks! I ended up refactoring/updating the example to not use `depends_on` at all, so this sentence kind of goes away anyway. Thank you!